plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/AmazonS3Fixture.java

```diff
@@ -113,23 +153,21 @@ protected Response handle(final Request request) throws IOException {
     }
 
     public static void main(final String[] args) throws Exception {
-        if (args == null || args.length != 3) {
-            throw new IllegalArgumentException(
-                "AmazonS3Fixture <working directory> <bucket for permanent creds> <bucket for temporary creds>");
+        if (args == null || args.length != 1) {
+            throw new IllegalArgumentException("AmazonS3Fixture <basedir");
         }
-
-        final AmazonS3Fixture fixture = new AmazonS3Fixture(args[0], args[1], args[2]);
+        final AmazonS3Fixture fixture = new AmazonS3Fixture(args[0]);
         fixture.listen();
     }
 
     /** Builds the default request handlers **/
-    private static PathTrie<RequestHandler> defaultHandlers(final Map<String, Bucket> buckets) {
+    private PathTrie<RequestHandler> defaultHandlers(final Map<String, Bucket> buckets) {
         final PathTrie<RequestHandler> handlers = new PathTrie<>(RestUtils.REST_DECODER);
 
         // HEAD Object
         //
         // https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectHEAD.html
-        objectsPaths("HEAD /{bucket}").forEach(path ->
+        objectsPaths(AUTH + " HEAD /{bucket}").forEach(path ->
```

Comment:
Nit: how about putting the space into `AUTH`/`NONAUTH` rather than including it in all these strings?